backup/slack_service.py

```diff
@@ -1,13 +1,29 @@
 import os
 
-from slack import WebClient
-from dotenv import find_dotenv, load_dotenv
+import slack
 
 
-load_dotenv(find_dotenv())
+class FakeWebClient:
```

Comment:
The fake client makes testing/developing locally a breeze. We don't have to modify anything and callers don't need to know if they are dealing with a fake client or real client. the `make_slack_client` function handles that for us.

Reply:
Responded to this in the previous message.

If the fake client is only used when testing, then we should just mock it instead of having to maintain two similar classes that does almost the same thing.

Reply:
I've removed the fake client. The test now uses mock.